### DISCUSSION TITLE
Will Wazuh be getting updated on SO?

### DISCUSSION THREAD

**Discussion by szym3n** [Security-Onion-Solutions/securityonion#8209] [General] [answered]
Hi!

I'm currently utilizing Wazuh for a number of FIM/OSSEC related operations on our network.
SO Is still running version 3.13 of Wazuh, but version 4.3 is already out.

Are there any plans on updating Wazuh to a later version any time soon?

Thanks!
Szymon

**Comment by UMHB-InfoSec** [comment]
I know it is in the priority backlog (https://github.com/Security-Onion-Solutions/securityonion/issues/1748)

To the SO team: I know you are all working on what you know to be important and I respect that; however please allow me to agree with @szym3n and request Wazuh at your earliest convenience.

**Reply by gitmffonseca** [reply]
It's already over a year....!
I really appreciate your efforts to give to SO2 what it is, but this is a major feature that can leverage your work even further!
Thank you!

**Reply by Jacob-Tate** [reply]
Whats the difference between 3.13 and 4.3? I know some of the updates have been backported by the Wazuh team.

**Comment by weslambert** [comment, marked as answer]
Status updates about Wazuh can be obtained through the following issue: https://github.com/Security-Onion-Solutions/securityonion/issues/1748

**Reply by szym3n** [reply]
Thank you Wes!

Is 4.1.5 the 'approved' version for upgrade? Do you know if there have been any issues found with the latest 4.3.5 version, or has that just not been investigated yet?

Also wanted to say I've had a very good experience with your support team during our professional engagements last year, so thank your team for that :)

Szymon
